### ISSUE TITLE
URL not long enough

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kiniaru/mobile-rss-reader#12]
```
What steps will reproduce the problem?
1. Add new feed
1. Enter URL 
http://newsrss.bbc.co.uk/rss/newsonline_uk_edition/technology/rss.xml as 

What is the expected output? Should have allowed the input.
What do you see instead? Got an error.


What version of the product are you using? Latest version.  On what 
operating system?
Using J2ME both Sun emulator and Sony Ericsson w710

Please provide any additional information below.

I have made my own fix to increase URL to 256 and recompiled and it works 
fine.
```

Original issue reported on code.google.com by `ibunto...@gmail.com` on 18 Jul 2007 at 7:32

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 1.5
```

Original comment by `ibunto...@gmail.com` on 6 Sep 2007 at 1:37
- Added labels: **Milestone-Release1.5**, **Priority-High**
- Removed labels: **Milestone-Release1.0**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
I have not heard any other complaints, so I'm saying that this is fixed.
```

Original comment by `ibunto...@gmail.com` on 23 Dec 2008 at 8:00
- Changed state: **Verified**
